packages/ember-htmlbars/lib/main.js

```diff
@@ -79,7 +67,6 @@ registerHelper('textarea', textareaHelper);
 registerHelper('collection', collectionHelper);
 registerHelper('each', eachHelper);
 registerHelper('unbound', unboundHelper);
-registerHelper('concat', concat);
```

Comment:
This line of code should have been removed in a previous commit. The concat hook doesn't have the right signature to be a helper anyways, see https://github.com/emberjs/ember.js/blob/master/packages/ember-htmlbars/lib/hooks/concat.js.